ui/App/ProfileScreen/ProjectCardSquare.svelte

```diff
@@ -27,6 +27,16 @@
     flex-direction: column;
     cursor: pointer;
   }
+
+  .project-card:hover {
+    background-color: var(--color-foreground-level-1);
+  }
+
+  .project-card:active {
+    transition: transform 0.1s ease-in-out;
+    transform: scale(0.95);
+  }
```

Comment:
Could we just use the global `.button-transition` class on the respective element?

Reply:
No, cause it also does the opacity thing, which we don't want here. What I could do is make a transition variable and use it in both places.

Reply:
Ok, never mind then.